rr_common/launch/demo_obstacle_follower.launch

```diff
@@ -1,14 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <launch>
+    <arg name="topic_from_controller" value="/control_effort"/>
+    <arg name="topic_from_plant" value="/state"/>
+    <arg name="setpoint_topic" value="/setpoint"/>
```

Comment:
Add `follower/` to the front of the topic names so that these topics won't interfere with other launch files where we also use the pid package.